science/2021-07-08-science-blog-2/index.md

```diff
@@ -54,7 +54,7 @@ Among other things, the event-driven user survey (EDUS) demonstrated:
 - Most of the respondents (61.6%) finished school with qualifications that provide entry to university or to a technical college
 - The majority of respondents live in urban areas and in western Germany
 - The majority of respondents followed typical social distancing and hygiene rules to protect themselves from infection
-- The vast majority of respondents who received a red warning (83.1%) subsequently took a COVID-19 test
+- The vast majority of respondents who received a red warning (83.1%) subsequently want to take a COVID-19 test
```

Comment:
That should be "wanted to take" since this is already a finished situation in the past. It only makes sense to take a test very soon after receiving a red warning, not weeks or months later.

Reply:
Okay, thanks for your suggestion.